content/javascript/concepts/dates/terms/toDateString/toDateString.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'toDateString()'
+Description: 'returns a string representing the date portion of a Date instance' 
+Subjects: 
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`toDateString()`** method of Date instances returns the date portion of a Date object as a string.
+
+## Syntax
+```pseudo
+const d = new Date('January 1, 2000 00:00')
+d.toDateString()
+```
+
+## Example
+```js
+const date = new Date("December 5, 2023 8:53")
+let  dateString= date.toDateString();
+console.log(dateString)
+// Output: Tue Dec 05 2023
```

Comment:
```suggestion
```
Add a output block below this code snippet with shell tag. Like this:
```shell
output
```
<img width="98" alt="image" src="https://github.com/Codecademy/docs/assets/94860622/c3898615-b0d4-4041-b5e3-9e4fba3318b6">